rtc-video-quality/encoder_commands.py

```diff
@@ -16,6 +16,7 @@
 import os
 import subprocess
 import tempfile
+from binary_vars import *
```

Comment:
Use import statements for packages and modules only, not for individual classes or functions.
http://google.github.io/styleguide/pyguide.html#22-imports